problems/removeVowels.js

```diff
@@ -6,8 +6,17 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+function removeVowels(str) {
+    let vowels = "AEIOUaeiou"
+    let newStr = ""
+    for(let i = 0; i < str.length; i++) {
+        if(vowels.includes(str[i])) {
```

Comment:
if you seek the negative !vowels.includes(str[i]) then you can use just the if, instead of having an else.